src/__tests__/lib/command/smartthings-command.test.ts

```diff
@@ -0,0 +1,161 @@
+import log4js from 'log4js'
+
+import { CLIConfig, loadConfig } from '../../../lib/cli-config.js'
+import { smartThingsCommand } from '../../../lib/command/smartthings-command.js'
+import { buildDefaultLog4jsConfig, loadLog4jsConfig } from '../../../lib/log-utils.js'
+
+
+jest.mock('log4js')
+jest.mock('../../../lib/cli-config.js')
+jest.mock('../../../lib/log-utils.js')
+jest.mock('../../../lib//table-generator')
+
+
+describe('smartThingsCommand', () => {
+	const defaultLogConfig = { config: 'default' } as unknown as log4js.Configuration
+	const buildDefaultLog4jsConfigMock = jest.mocked(buildDefaultLog4jsConfig).mockReturnValue(defaultLogConfig)
+	const logConfig = { config: 'final' } as unknown as log4js.Configuration
+	const loadLog4jsConfigMock = jest.mocked(loadLog4jsConfig).mockReturnValue(logConfig)
+	const log4jsConfigureMock = jest.mocked(log4js.configure)
+	const traceMock = jest.fn()
+	const loggerMock = {
+		trace: traceMock,
+	} as unknown as log4js.Logger
+	const log4jsGetLoggerMock = jest.mocked(log4js.getLogger).mockReturnValue(loggerMock)
+	const cliConfig = { profile: {} } as CLIConfig
+	const loadConfigMock = jest.mocked(loadConfig).mockResolvedValue(cliConfig)
+
+	it('works', async () => {
```

Comment:
LOL at this test name!

Reply:
😂  Oops...that was not meant to stay in! I almost always start with `works` for my first test and fix it. I think I'll leave it until I flesh these tests out more though.